### PR TITLE
Fixes for @ckeditor/ckeditor5-paragraph

### PR DESCRIPTION
**Pull request by fedemp**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
- https://github.com/ckeditor/ckeditor5/blob/72f503139a8fdf40b41cf26960a94b725f409338/packages/ckeditor5-paragraph/src/insertparagraphcommand.js#L34
- https://github.com/ckeditor/ckeditor5/blob/72f503139a8fdf40b41cf26960a94b725f409338/packages/ckeditor5-paragraph/src/paragraph.js#L44-L45
- https://github.com/ckeditor/ckeditor5/blob/72f503139a8fdf40b41cf26960a94b725f409338/packages/ckeditor5-paragraph/src/paragraphbuttonui.js#L36
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/ckeditor__ckeditor5-paragraph/src/paragraphcommand.d.ts [hunk 1]
```diff
@@ -17,4 +17,6 @@ export default class ParagraphCommand extends Command {
      * will be turned to paragraphs.
      */
     execute(options?: { selection?: Selection | DocumentSelection | undefined }): void;
+
+    refresh(): void;
```
Comment (MysteryBlokHed) [inline types/ckeditor__ckeditor5-paragraph/src/paragraphcommand.d.ts:21]: Is defining `refresh()` here needed if it exists on the base `Command` class?

Comment (fedemp) [reply types/ckeditor__ckeditor5-paragraph/src/paragraphcommand.d.ts]: No, it's not. I've removed it now.

**Review by MysteryBlokHed — APPROVED** [review]
LGTM!

**Comment by fedemp** [comment]
Ready to merge

**Pull request merged**